compiler/rustc_resolve/src/lib.rs

```diff
@@ -2006,6 +2006,23 @@ impl<'a> Resolver<'a> {
         }
         self.main_def = Some(MainDefinition { res, is_import, span });
     }
+
+    pub fn export_check(&self, binding: &NameBinding<'a>) -> Option<def::Res<!>> {
```

Comment:
```suggestion
    pub fn is_reexport(&self, binding: &NameBinding<'a>) -> Option<def::Res<!>> {
```
this also needs a comment, something like
```
// Items that go to reexport table encoded to metadata and visible through it to other crates.
```